README.md

```diff
@@ -59,34 +59,44 @@ To run this sample:
     ```
 
 5. You're all set! Open a browser window to [http://localhost:3000](http://localhost:3000).
+#### Note
+Use chrome browser if you're running the mock-okta server, to test the sample.
+
+## Using your own Okta org
+You can create your own Okta org by signing up for a [free Developer Account](#https://developer.okta.com/signup/)
+
+If you'd like to test this sample against your own Okta org, navigate to the Okta Developer Dashboard and follow these steps:
+
+1. Create a new **Web** application by clicking **Add Application** and selecting **Web** from the *Applications* page.		
+2. After accepting the default configuration, click **Done** to redirect back to the *General Settings* of your application.		
+3. Copy the **Client ID** and **Client Secret**, as it will be needed for the client configuration.
+4. Finally, navigate to `https://{yourOktaDomain}.com/api/v1/authorizationServers/default` to see if the [Default Authorization Server](https://developer.okta.com/docs/api/resources/oauth2.html#using-the-default-authorization-server) is setup. If not, [let us know](mailto:developers@okta.com).
+Then, replace the *oidc* settings in `.samples.config.json` to point to your new app:
+```javascript
+// .samples.config.json
+{
+  "oidc": {
+    "oktaUrl": "https://{{yourOktaDomain}}.com",
+    "issuer": "https://{{yourOktaDomain}}.com/oauth2/default",
+    "clientId": "{{yourClientId}}",
+    "clientSecret": "{{yourClientSecret}}",
+    "redirectUri": "http://localhost:3000/authorization-code/callback"
+  }
+}
+```
+
+
+Stop the mock-okta server and Express server if you're running it.
+ 
+Open a new terminal window, and run the Express server.
+   
+   ```bash
+   # Starts the sample back end on http://localhost:3000
+   [samples-js-angular-1]$ npm start
+   ```
+   
+You're all set! Open a browser window to [http://localhost:3000](http://localhost:3000).
 
-## Using a different back end
```

Comment:
We still need to keep this. Can you create a new `README-v1.md` file that contains the "old README"? Then we can link to it in our callout above.

Reply:
I have linked the older README at the top, where I ask users to use previous version if they want to use different back-ends